_layouts/home.html

```diff
@@ -350,9 +350,6 @@ <h5>Works with the tools you already use</h5>
             <div class="col-md-4 p-1 p-lg-2">
```

Comment:
Should we move the other items in this row over, so they're centered?

Reply:
sounds good!